packages/tasks/ios/install.js

```diff
@@ -0,0 +1,43 @@
+const process = require('process');
```

Comment:
Does this file is really used? Seems that we do the same logic in the template's tasks?

Reply:
Indeed, I'll reuse the tasks in `.template.js`.